tests/Integration/Form/Location/LocationTypeTest.php

```diff
@@ -41,13 +42,30 @@ protected function tearDown(): void
 
     public function testBuildForm(): void
     {
-        /* @todo This test is incomplete. */
-        self::markTestIncomplete();
+        $type = new Location();
+
+        $formData = [
+            'name' => 'here',
+            'address' => 'there, obv',
+        ];
+
+        /** @var FormFactoryInterface $formfactory */
+        $formfactory = self::getContainer()->get('form.factory');
+        $form = $formfactory->create(LocationType::class, $type, ['csrf_protection' => false]);
+
+        $form->submit($formData);
+        self::assertTrue($form->isSynchronized());
+        self::assertTrue($form->isSubmitted());
+        self::assertTrue($form->isValid());
     }
 
     public function testConfigureOptions(): void
     {
-        /* @todo This test is incomplete. */
-        self::markTestIncomplete();
+        /** @var MockObject&OptionsResolver $resolver */
```

Comment:
Same thing here, this is the way we currently do things, but I'd suggest something like...
```php
$resolver = new OptionsResolver(); // just use the real deal, it's an integration test anyways
$this->locationType->configureOptions($resolver);
$opts = $resolver->resolve([]);

self::assertSame($opts['data_class'], Location::class);
```

Reply:
you're right, this is way nicer! and we can yeet yet another mock